.env.example

```diff
@@ -254,6 +254,8 @@ AZURE_APP_SECRET=
 # Sentiment analysis model file path
 SENTIMENT_FILE_PATH=
 
+## CLEARBIT INTEGRATION
+CLEARBIT_API_KEY=
```

Comment:
Let us move this to installation config only and not use env variables.

Reply:
@sojan-official Please assist @scmmishra for the changes required.

Reply:
let's remove this from the env variable altogether. no need to expose it here.